ci/lint

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+SUCCESS=true
+
+# Ensure Licenses are present
+if licenser verify -r .; then
+    echo "Licenses are present on all recognised files."
+else
+    echo "Licenses are missing."
+    SUCCESS=false
```

Comment:
nit: I prefer early exits, but since current impl has the advantage that this can show full lint errors, it's okay.